src/app/pages/send/send-crypto-asset-form/components/recipient-select/recipient-select.layout.tsx

```diff
@@ -0,0 +1,46 @@
+import { ReactNode, useRef } from 'react';
+
+import { Fade, Stack, color } from '@stacks/ui';
+
+import { useOnClickOutside } from '@app/common/hooks/use-onclickoutside';
+
+import { RecipientSelectItem } from './recipient-select-item';
+
+interface RecipientSelectLayoutProps {
+  children: ReactNode;
+  isVisible: boolean;
+  onSetIsSelectVisible(value: boolean): void;
+  selectedItem: number;
+}
+export function RecipientSelectLayout(props: RecipientSelectLayoutProps) {
+  const { children, isVisible, onSetIsSelectVisible, selectedItem } = props;
+  const ref = useRef<HTMLDivElement | null>(null);
+
+  useOnClickOutside(ref, () => onSetIsSelectVisible(false));
+
+  return (
+    <Stack _hover={{ cursor: 'pointer' }} maxHeight="base-loose">
+      <RecipientSelectItem index={selectedItem} onSelectItem={() => onSetIsSelectVisible(true)} />
+      <Fade in={isVisible}>
+        {styles => (
+          <Stack
+            bg={color('bg')}
+            borderRadius="8px"
+            boxShadow="high"
+            flexDirection="column"
+            minWidth="100px"
+            overflow="hidden"
+            p="extra-tight"
+            position="absolute"
+            ref={ref}
+            style={styles}
+            top="40px"
+            zIndex={9999}
+          >
+            {children}
+          </Stack>
+        )}
+      </Fade>
+    </Stack>
```

Comment:
I know you're reluctant to use, and not suggesting to change rn, but a dropdown component would be easier implementation, and have better ux.

Consider UX behaviours: advanced mouse controls, keyboard controls (esc to close, arrow keys to select) etc.